src/platform/native/Core.zig

```diff
@@ -1039,6 +1039,12 @@ pub inline fn wakeMainThread(self: *Core) void {
     glfw.postEmptyEvent();
 }
 
+// May be called from any thread.
+pub fn nativeWindowCocoa(self: *Core) ?*anyopaque {
```

Comment:
I don't think this should return an optional pointer, right?

Reply:
I thought that maybe it should be okay to call this without even calling init without panic, but you are right. It should be illegal the same way as it's illegal to call this on Windows, for example. Fixed